pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.pytest.ini_options]
+testpaths = ["tests"]
```

Comment:
@KristijanArmeni a thought -- what do you think about allowing each analyzer to have its own test folder so that each module is self contained? How tricky would it be to configure something like that?

Reply:
I don't think it should be too tricky, as it is one of the two supported layouts by pytest. I can't say I see an obvious advantage for one or the other layout atm. I guess once you have many tests, it becomes more manageable to put them closer to code?

Separate test folder:
> Putting tests into an extra directory outside your actual application code might be useful if you have many functional tests or for other reasons want to keep tests separate from actual application code (often a good idea)

Tests as part of modules:  
> Inlining test directories into your application package is useful if you have direct relation between tests and application modules and want to distribute them along with your application

See more under 'Tests layout' here: https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html